src/agent/onefuzz-task/src/tasks/coverage/generic.rs

```diff
@@ -174,7 +173,7 @@ impl CoverageTask {
             context.save_and_sync_coverage().await?;
         }
 
-        info!("report coverage");
+        info!("report final coverage");
```

Comment:
I'm not sure what this comment is meant to convey. Later in the function we still poll the queue for inputs, so there could still be more coverage, right?